PayPal/src/main/java/com/braintreepayments/api/PayPalRequest.java

```diff
@@ -27,6 +27,7 @@ public abstract class PayPalRequest implements Parcelable {
     static final String BILLING_AGREEMENT_DETAILS_KEY = "billing_agreement_details";
     static final String DESCRIPTION_KEY = "description";
     static final String PAYER_EMAIL_KEY = "payer_email";
+    static final String PHONE = "phone";
```

Comment:
```suggestion
    static final String PHONE_KEY = "phone";
```
Adding key suffix to match convention.

Reply:
Good catch! just updated the name